modules/objc/examples/code_snippets/SampleCodeTest.m

```diff
@@ -2334,183 +2277,111 @@ func mystopfunc() {
+
+- (CBLTLSIdentity*) fMyGetCert {
+    NSError *error = nil;
+    CBLReplicatorConfiguration *config; // FIXME: Are we sure? this is Replicator config?
+    // tag::p2p-tlsid-tlsidentity-with-label[]
+    // tag::p2p-tlsid-check-keychain[]
+    // Check if Id exists in keychain and if so, use it
+    CBLTLSIdentity *identity = [CBLTLSIdentity identityWithLabel:@"doco-sync-server" error:&error]; // <.>
+
+    // end::p2p-tlsid-check-keychain[]
+    config.authenticator = [[CBLClientCertificateAuthenticator alloc] initWithIdentity:identity]; // <.>
+
+    // end::p2p-tlsid-tlsidentity-with-label[]
+    
+    // tag::p2p-tlsid-check-bundled[]
+    // CREATE IDENTITY FROM BUNDLED RESOURCE IF FOUND
+    
+    // Check for a resource bundle with required label to generate identity from
+    // return nil identify if not found
+    NSString *path = [[NSBundle mainBundle] pathForResource:@"doco-sync-server" ofType:@"p12"];
+    NSData *data = [NSData dataWithContentsOfFile:path
+                                              options:0
+                                                error:&error];
+    if (!data)
+        return nil;
+    
+    // end::p2p-tlsid-check-bundled[]
+    
+    // tag::p2p-tlsid-import-from-bundled[]
+    // Use SecPKCS12Import to import the contents (identities and certificates)
+    // of the required resource bundle (PKCS #12 formatted blob).
+    //
+    // Set passphrase using kSecImportExportPassphrase.
+    // This passphrase should correspond to what was specified when .p12 file was created
+    CFArrayRef result = NULL;
+    NSDictionary *options = @{ (id)kSecImportExportPassphrase:@"couchbase" };
+    OSStatus status = SecPKCS12Import((__bridge CFDataRef)data, (__bridge CFDictionaryRef)options, &result);
+    if (status != errSecSuccess) {
+        NSLog(@"Failed to import data from provided with error :%d ", (int)status);
+        return nil;
+    }
+    
+    NSArray *importedItems = (NSArray*)CFBridgingRelease(result);
+    NSDictionary *item = importedItems[0];
+    
+    // Get SecIdentityRef representing the item's id
+    SecIdentityRef identityRef = (__bridge SecIdentityRef) item[(id)kSecImportItemIdentity];
+    
+    // Get Id's Private Key, return nil id if fails
+    SecKeyRef privateKey;
+    status = SecIdentityCopyPrivateKey(identityRef, &privateKey);
+    if (status != errSecSuccess) {
+        NSLog(@"Failed to import private key from provided with error :%d ", (int)status);
+        return nil;
+    }
+    CFAutorelease(privateKey);
+    
+    // Get all relevant certs [SecCertificate] from the ID's cert chain using kSecImportItemCertChain
+    NSArray *certChain = item[(id)kSecImportItemCertChain];
+    
+    // Return nil Id if errors in key or cert chain at this stage
+    if (!certChain || !privateKey)
+        return nil;
+    
+    // end::p2p-tlsid-import-from-bundled[]
+    
+    // tag::p2p-tlsid-store-in-keychain[]
+    // FIXME: Use the same tag as tag: [create-self-signed-cert]
```

Comment:
p2p-tlsid-store-in-keychain[] not required.  if user wants to save the identity, they can call the createIdentity or importIdentity sample.

refer: https://github.com/couchbase/docs-couchbase-lite/pull/635#discussion_r746988320

Reply:
Noted. This page is currently redundant and may be removed entirely.